### PR TITLE
WIP:Feat/landing page

### PR DESCRIPTION
**Pull request by CJ-Rose**

### CODE REVIEW

**Comment by vivianjeng** [comment]
Hi @CJ-Rose 
we are using `yarn` for cloudflare building
so please run `yarn install` and `yarn build` in your local
thank you!

File: mopro-web/docusaurus.config.ts [hunk 1]
```diff
@@ -51,74 +51,111 @@ const config: Config = {
...
+        // },
         {
-          href: 'https://github.com/zkmopro/mopro',
-          label: 'GitHub',
+          type: 'html',
           position: 'right',
+          value: 
+            `<div class="custom-navbar-link">
+              <a href="https://github.com/zkmopro/mopro" target="_blank">Github</a
+              ><img src="img/link_arrow.svg"/>
```
Comment (vivianjeng) [inline mopro-web/docusaurus.config.ts:89]: when I am in the `docs/` the `link_arrow` image is not found

Comment (vivianjeng) [reply mopro-web/docusaurus.config.ts:89]: 截圖 2024-06-03 下午5 28 14

File: mopro-web/src/pages/index.module.css [hunk 2]
```diff
@@ -3,21 +3,39 @@
  * and scoped locally.
  */
 
-.heroBanner {
-  padding: 4rem 0;
-  text-align: center;
-  position: relative;
-  overflow: hidden;
+h1 {
```
Comment (vivianjeng) [inline mopro-web/src/pages/index.module.css:6]: The h1 width seems like not enough for the titles
I think the `width: 50%` can be removed?
截圖 2024-06-03 下午5 29 21

File: mopro-web/src/css/custom.css [hunk 3]
```diff
@@ -5,26 +5,94 @@
...
+  --ifm-color-primary-light: #000;
+  --ifm-color-primary-lighter: #000;
+  --ifm-color-primary-lightest: #000;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
+
+a:active, a:visited, a:hover {
+  color: white;
+  text-decoration: none;
+}
```
Comment (vivianjeng) [inline mopro-web/src/css/custom.css:39]: It is hard to distinguish between text and links.
maybe we can discuss with Chiali
截圖 2024-06-03 下午5 31 44

**Review by vivianjeng — COMMENTED** [review]
Thank you CJ!
It looks really good and match the design so well.
I think some commented out codes can be removed before merged

File: mopro-web/docusaurus.config.ts (hunk 1 above)
Comment (CJ-Rose) [reply mopro-web/docusaurus.config.ts:89]: ✅ fixed

File: mopro-web/src/css/custom.css (hunk 3 above)
Comment (CJ-Rose) [reply mopro-web/src/css/custom.css]: - updated links to orange color, white on hover.
- 'p' texts in docs updated to light yellow.
- Docs sidebar links are yellow, orange on hover. docusaurus is hiding the way to change the color for "active" state of sidebar items, i haven't found it yet 🫤

**Pull request merged**